### ISSUE TITLE
Update encrypt and decrypt account utils with SDK implementation

### ISSUE THREAD

**Issue by ikem-legend** [LiskHQ/lisk-desktop#4342]
### Description

The `encryptAccount` and `decryptAccount` utils were implemented using mocks and need to be implemented properly when the functionality from SDK is ready

### Acceptance Criteria

- Change the implementation of the `encryptAES256GCMWithPassword` and `decryptAES256GCMWithPassword` functions to be imported from `lisk-client`
- Ensure the end to end flow after integration
  - Users should be able to create new recovery phrase
  - Users should be able to use existing recovery phrase
  - Users should be able to backup recovery phrase 
  - In short all the account management features should work without mocks
- Update unit and e2e tests

**Comment by ManuGowda** [comment]
Fix the following scenarios:

- edit account
- desing curve is not matching to figma
- the wallet icons missing
- name length validation missing (we can use the same requirement from delegate name registrtion)
- switch account close button is missing
- create one is not working
- remove account UI is not working according to figma desing
- remove account doesn't remove the account from local storage
- choose account navigation doesn't redirect to wallet page

**Comment by ManuGowda** [comment]
api change reference https://github.com/LiskHQ/lisk-sdk/issues/6710

**Comment by ManuGowda** [comment]
The following issue needs separate issue:
- [x] Choose account navigation doesn't redirect to wallet page
- [x] We need to prettify the encrypted account json shown on upload by file add account method.

**Comment by ManuGowda** [comment]
The missing tasks will be implemented as part of https://github.com/LiskHQ/lisk-desktop/issues/4409
